lib/airtable.js

```diff
@@ -1,19 +1,29 @@
-import { Airtable as AirtableOptions } from "./options.js"
+import { AirtableNwif as NwifOptions, AirtableScif as ScifOptions } from "./options.js"
 import Airtable from "airtable"
 
-const getClient = new Promise(resolve => {
-	const client = new Airtable({ apiKey: AirtableOptions.apiToken })
-	resolve(client)
-})
+const existingClients = {};
 
-const getBase = async () => {
-	const client = await getClient
-	const base = client.base(AirtableOptions.baseID)
+const getClient = async (options) => {
+	if (existingClients[options.baseID]) {
+		return existingClients[options.baseID]
+	}
+	const client = new Airtable({ apiKey: options.apiToken })
+	existingClients[options.BaseID] = client
```

Comment:
I think this should be `options.baseID`, not `options.BaseID`

Reply:
It sure should, that cache does nothing right now. Good eye!